pkg/operator/ceph/object/controller_test.go

```diff
@@ -334,7 +335,7 @@ func TestCephObjectStoreController(t *testing.T) {
 		clientset := test.New(t, 3)
 		c := &clusterd.Context{
 			Executor:      executor,
-			RookClientset: rookclient.NewSimpleClientset(),
+			RookClientset: rookfake.NewSimpleClientset(),
```

Comment:
Looks like  rookclient and rookfake are the same import so we could remove one of them.

Reply:
Yeah, though I'll think it is better to change this and other "linter errors" in a separate PR.